src/pages/studyView/UserSelections.tsx

```diff
@@ -60,6 +63,8 @@ export interface IUserSelectionsProps {
     caseListNameSet: { [key: string]: string };
     removeGenomicProfileFilter: (value: string) => void;
     removeCaseListsFilter: (value: string) => void;
+    removeSampleTreatmentsFilter: (outerIndex: number, innerIndex: number) => void;
```

Comment:
what's the meaning of the terminology `outerIndex` and `innerIndex`?

Reply:
Treatments filters are a 2d array. `outer` refers to the outer array, and `inner` refers to the inner. I'm open to suggestions if you can think of better names.
It is a little confusing because of the structure of the filter. I wanted to express the relationship between filters in the type itself, so I opted for this:
```typescript
patientTreatmentFilters: AndedPatientTreatmentFilters;
type AndedPatientTreatmentFilters = {
    'filters': Array < OredPatientTreatmentFilters >
};
type OredPatientTreatmentFilters = {
    'filters': Array < PatientTreatmentFilter >
};
```
This makes the relation between filters much clearer, but at the expense of making the fact that this is a 2d array a little less clear.

Reply:
I see. So what you have is an AND of OR's, aka conjunctive normal form. Maybe `andedIndex` instead of `outerIndex` and `oredIndex` instead of `innerIndex`?